automation/project/Dependencies.scala

```diff
@@ -45,6 +45,11 @@ object Dependencies {
       excludeApacheHttpClient,
       excludeGoogleJsr305,
       excludeJacksonCore),
+    "com.google.apis" % "google-api-services-sourcerepo" % s"v1-rev21-1.23.0" excludeAll (
```

Comment:
Just added for the `CloudSourceRepositoriesScopes` class.

Reply:
can you upgrade workbench-service-test version to the latest `val serviceTestV = "0.12-1a00943"`? Credential scopes is newly required parameter in `UserAuthToken`. I need `createDownloadDirectory()` in new version of workbench-service-lib.

Reply:
I updated to the latest `workbench-service-test`.